iree/samples/simple_embedding/simple_embedding.c

```diff
@@ -32,6 +29,10 @@
 #include "iree/samples/simple_embedding/simple_embedding_test_bytecode_module_c.h"
 #endif
 
+// A function to create the HAL device from the different backend targets.
+// The HAL device is returned based on the implementation.
+extern iree_status_t create_sample_device(iree_hal_device_t** device);
```

Comment:
could document ownership here (the device returned must be released by the caller)

Reply:
Done.